src/fmu/sumo/explorer/objects/_child_collection.py

```diff
@@ -107,6 +107,16 @@ def vertical_domain(self) -> List[str]:
     async def vertical_domain_async(self) -> List[str]:
         """List of unqiue object vertical domain"""
         return await self._get_field_values_async("data.vertical_domain")
+   
+    @property
+    def contents(self) -> List[str]:
+        """List of unique contents"""
+        return self._get_field_values("data.content.keyword")
+
+    @property
+    async def contents_async(self) -> List[str]:
```

Comment:
Can it be confusing that these say `contentS`, while the metadata is `content`?

Reply:
Could be.. looks like we've gone with plurals for most of the other properties (names, tagnames, iterations, realizations, stages). Maybe we should switch strategy completely and add a method for getting unique values, so it would be something like `surfaces.unique("data.content")`